test/smoke_test.rb

```diff
@@ -33,6 +35,25 @@ def test_it_finds_all_movies
     assert_equal Movie.all.size, search.result.size
   end
 
+  def test_scope_filter
+    search = MovieSearch.new({'search' => { 'classics' => 'classics'}})
+    assert_equal 6, search.result.size
+    search = MovieSearch.new({'search' => { 'classics' => 'non_classics'}})
+    assert_equal 34, search.result.size
+    search = MovieSearch.new({'search' => { 'classics' => ['non_classics', 'classics']}})
```

Comment:
Line is too long. [89/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.